webpack/assets/javascripts/react_app/components/bookmarks/Bookmark.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import URI from 'urijs';
+import { MenuItem } from 'react-bootstrap';
+
+function followLink() {
+  let uri = new URI(window.location.href);
+
+  uri.setSearch('search', this.trim());
+  /* eslint-disable no-undef */
+  Turbolinks.visit(uri.toString());
```

Comment:
Is it worth adding this dependency to turbolinks if we are going to remove it?

Reply:
yes, I don't think we are going to remove it any time soon, also its consistent with the rest of the page (e.g. when you click on search, or when you use the pagination controllers) - I can't see why clicking on the bookmark should do anything different.